.github/workflows/trigger_azdo_ci.yml

```diff
@@ -1,7 +1,10 @@
 name: Trigger AzDO CIs
 
 on:
-  workflow_call:
+  workflow_run:
```

Comment:
Note: I think we could also trigger these only on CI success (everything through the system_tests passing). See [these GitHub docs ](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow) for how that would be enabled.

For now, I'm keeping it as it was to always trigger the AzDo CIs upon GitHub CI completion. @csjall , does that sound reasonable?

Reply:
Agreed. My main concern was with the timing. The adhoc export pipeline simply grab the latest artifacts. As long as we trigger the pipeline on completion of the CI I think it should be fine. We will look to system test result before we decide to release a specific release.